flask_app.py

```diff
@@ -0,0 +1,148 @@
+"""
+I made a simple flask app that creates a SQLAlchemy database that stores simple information like the email, password and token.
+
+To make it work locally you have to install quite some modules.
+http://127.0.0.1:5000/
+
+The database is now filed with two user testcases, what you can do is delete the site.db from the data / database directory to test the user creation for yourself
+
+You can adapt the code to use another database to store the user information in.
+"""
+
+import os
+from flask import Flask, request, render_template
+from flask_jwt_extended import (
+    JWTManager,
+    create_access_token,
+    jwt_required,
+    get_jwt_identity,
+)
+from flask_sqlalchemy import SQLAlchemy
+from flask_bcrypt import Bcrypt
+from flask_login import UserMixin
+from flask_migrate import Migrate
+import hashlib
+
+app = Flask(__name__)
+
+# this finds the route to store the database to the data / database directory
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
+
+db_dir = os.path.join(project_root, 'data', 'database')
+
+if not os.path.exists(db_dir):
+    os.makedirs(db_dir)
+
+app.config["SQLALCHEMY_DATABASE_URI"] = f"sqlite:///{os.path.join(db_dir, 'site.db')}"
```

Comment:
This should be an environment variable that's loaded in from a locally stored `.env` file at the repository root folder when someone runs the Flask app using [`python-dotenv`](https://pypi.org/project/python-dotenv/). It should be checked for within the `os.environ` dictionary (after setting up `python-dotenv`) in case it exists. Only if it doesn't would it make sense to use a fallback URL and create a new database.

Creating a `.env.example` would make sense here that people can rename locally to `.env` .